spring-cloud-square-retrofit-webclient/src/test/java/org/springframework/cloud/square/retrofit/webclient/WebClientRetrofitLoadBalancerTests.java

```diff
@@ -89,6 +111,12 @@ void testCustomWebClientBuilderPickeByRetrofitName() {
 				.isThrownBy(() -> testClientWithCustomWebClientBuilder.getHello().block());
 	}
 
+	@Test
+	void testCustomWebClientBuilderPickeByRetrofitNameWithoutLoadBalanced() {
```

Comment:
Please rename to `testCustomWebClientBuilderPickedByRetrofitNameWithoutLoadBalanced` (typo)